docs/Using-Fleet/REST-API.md

```diff
@@ -3540,7 +3540,11 @@ These API endpoints are used to automate MDM features in Fleet. Read more about
 - [Upload a bootstrap package](#upload-a-bootstrap-package)
 - [Get metadata about a bootstrap package](#get-metadata-about-a-bootstrap-package)
 - [Delete a bootstrap package](#delete-a-bootstrap-package)
-- [Download a bootstrap package](#download-a-bootstrap-package)
+- [Download a bootstrap package](#download-an-bootstrap-package)
+- [Upload an EULA file](#upload-an-eula-file)
```

Comment:
```suggestion
- [Upload a EULA file](#upload-a-eula-file)
```

Reply:
On second thought, I'm incorrect. In my head, I say "You-la", but pronouncing the acronym E-U-L-A should be "an" and not "a". 😵

Reply:
I originally used "a", and since this always trips me I did a google search 😅